packages/components/organisms/f-checkout/src/components/Notes.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+        <accordion
+            :id="noteTypeDeliveryOrRestaurant"
+            :title="$t(`userNote.${noteTypeDeliveryOrRestaurant}.${serviceType}.title`)">
```

Comment:
Is it worth having a computed property that handles all of these translations?
so
```
accordianTranslations () {
  return {
    id: $t(`userNote.${noteTypeDeliveryOrRestaurant}.${serviceType}.title`),
    placeholder: $t(`userNote.${noteTypeDeliveryOrRestaurant}.${serviceType}.placeholder`)
    ...
  }
}
```
Then these can just be `:id="accordianTranslations.id"`

Reply:
Accordion* 😬

Reply:
Is there a way to reference `$t` outside of the template @FayeCarter? 👀  I haven't seen any other occurrences of it.